IOS_NUMERIC_KEYPAD_ANALYSIS.md

```
@@ -0,0 +1,169 @@
+   - `TEL` → `inputmode="tel"` (already used)
+
+3. **Support both `inputmode` (modern) and `type="tel"` (fallback)**
+   - iOS Safari 13.0-14.x: Uses type="tel"
+   - iOS Safari 15.0+, other browsers: Uses inputmode
+
+4. **Pass through to actual input elements**
+   - StyledNumericInput: Add inputmode prop
+   - InputGroup (text): Add inputmode prop
+
+### Browser Support Matrix
+| Browser | inputmode | type="tel" | Notes |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)